test/hooks/validate-animal.test.js

```diff
@@ -0,0 +1,33 @@
+
+const assert = require('assert');
+const feathers = require('@feathersjs/feathers');
+const validateAnimal = require('../../src/hooks/validate-animal.js');
+
+describe('\'validate animal\' hook', () => {
+  let app;
+
+  beforeEach(() => {
+    app = feathers();
```

Comment:
I wonder why this doesn't use something like `this.app` (like app.test.js uses `this.server` on the before hook)